### ISSUE TITLE
MultilineRawStringIndentation false positive when using tab indentation

### ISSUE THREAD

**Issue by TWiStErRob** [detekt/detekt#5564]
Same as https://github.com/detekt/detekt/issues/5429

The original test case from #5429's OP wasn't implemented in #5453:
```
        @Test
        fun `don't raise multiline raw strings if all have the correct indentation - tabs`() {
            val TAB = "\t"
            val code = """
                val a = 
                ${TAB}$TQ
                ${TAB}${TAB}Hello world!
                ${TAB}${TAB}How are you?
                ${TAB}$TQ.trimIndent()
            """.trimIndent()
            subject.compileAndLint(code)
            assertThat(subject.findings)
                .isEmpty()
        }
```
if all the indentation levels are tabs, it miscalculates the expected by 3:
```
LintMessageDetailsSplitter.kt:65:11: The indentation should be 13 but it is 10. [MultilineRawStringIndentation]
LintMessageDetailsSplitter.kt:70:10: The indentation should be 12 but it is 9. [MultilineRawStringIndentation]
SuppressionGenerator.kt:16:7: The indentation should be 9 but it is 6. [MultilineRawStringIndentation]
```
When the code is indented by tabs, it's expected to have the indentation inside the raw string as tab too, because this rule is enforcing code indentation inside a raw string, therefore it's not part of the output.

**Comment by t-kameyama** [comment]
Indent size should be 1.

```kotlin
        @Test
        fun `don't raise multiline raw strings if all have the correct indentation - tabs`() {
            val code = """
                val a = 
                ${TAB}$TQ
                ${TAB}${TAB}Hello world!
                ${TAB}${TAB}How are you?
                ${TAB}$TQ.trimIndent()
            """.trimIndent()
            val subject = MultilineRawStringIndentation(TestConfig(mapOf("indentSize" to 1)))
            subject.compileAndLint(code)
            assertThat(subject.findings)
                .isEmpty()
        }
```

**Comment by TWiStErRob** [comment]
Oooh, nice! Thank you, will try it.

**Comment by TWiStErRob** [comment]
Works as advertised, https://github.com/TWiStErRob/net.twisterrob.gradle/pull/335/commits/79ffdd6c85a94346c1f10f6bb246e4c88bfb6218

So this is not a false positive, just a non-conventional setup.
